Scheduler/feature_scheduler/maintel/fbs_config_sit_survey_block_t248.py

```diff
@@ -0,0 +1,190 @@
+    dec=-60.0,
+    sequence=["r_03", "i_06"],
+    nside=32,
+    nexp=1,
+    exptime=30.0,
+    visits=1,
+    camera_rot_limits=[-80.0, 80.0],
+    shadow_minutes=60.0,
+    max_alt=86.0,
+    ignore_obs="DD",
+    slewtime_weight=3.0,
+    stayfilter_weight=3.0,
+):
+    """A survey class for running field surveys.
+
+    Parameters
+    ----------
+    name : `str`  name of survey
+    RA : `float`
+        The RA of the field (degrees)
+    dec : `float`
+        The dec of the field to observe (degrees)
+    sequence : `list` [`str`]
+        The sequence of observations to take. (specify which filters to use).
+    nvisits : `dict` {`str`: `int`}
+        Dictionary of the number of visits in each filter.
+        Default of None will use a backup sequence of 20 visits per filter.
+        Must contain all filters in sequence.
+    exptimes : `dict` {`str`: `float`}
+        Dictionary of the exposure time for visits in each filter.
+        Default of None will use a backup sequence of 38s in u, and
+        29.2s in all other bands. Must contain all filters in sequence.
+    nexps : dict` {`str`: `int`}
+        Dictionary of the number of exposures per visit in each filter.
+        Default of None will use a backup sequence of 1 exposure per visit
+        in u band, 2 in all other bands. Must contain all filters in sequence.
+    ignore_obs : `list` [`str`] or None
+        Ignore observations with this string in the `scheduler_note`.
+        Will ignore observations which match subsets of the string, as well as
+        the entire string. Ignoring 'mysurvey23' will also ignore 'mysurvey2'.
+    survey_name : `str` or None.
+        The name to give this survey, for debugging and visualization purposes.
+        Also propagated to the 'target_name' in the observation.
+        The default None will construct a name based on the
+        RA/Dec of the field.
+    scheduler_note : `str` or None
+        The value to include in the scheduler note.
+        The scheduler note is for internal, scheduler, use for the purposes of
+        identifying observations to ignore or include for a survey or feature.
+    readtime : `float`
+        Readout time for computing approximate time of observing
+        the sequence. (seconds)
+    filter_change_time : `float`
+        Filter change time, on average. Used for computing approximate
+        time for the observing sequence. (seconds)
+    nside : `float` or None
+        Nside for computing survey basis functions and maps.
+        The default of None will use rubin_scheduler.utils.set_default_nside().
+    flush_pad : `float`
+        How long to hold observations in the queue after they
+        were expected to be completed (minutes).
+    """
+    # Define the extra parameters that are used in the greedy survey. I
+    # think these are fairly set, so no need to promote to utility func kwargs
+
+    survey_detailers = [
+        detailers.TrackingInfoDetailer(science_program=name),
```

Comment:
I think you can pass `science_program` to the `FieldSurvey` and then you don't need the detailer.

Reply:
It's even preferable to just pass "science_program" as a kwarg to the FieldSurvey, which will indeed trigger the TrackingInfoDetailer, but then it also will also be  saved in the survey as an attribute (might permit easier debugging at this stage). 

```
        survey = FieldSurvey(
            basis_functions,
            ra,
            dec,
            sequence=sequence,
            exptimes=exptimes,
            nexps=nexps,
            nvisits=nvisits,
            nside=nside,
            ignore_obs=ignore_obs,
            detailers=survey_detailers,
            survey_name=name,
            science_program=name,
        )
 ```
 
 And actually, I don't know what downstream processing wants for things like target_name, if they'd like to have the different observations in the sequence have a different target_name .. I would have thought, on some level that having the observations of the same thing (but with different dithers) have the same name would be preferable, but it's hard to know at this stage of things. You *could* also set target_name = science_program.split("_")[0] if you wanted the target name to be the same just BLOCK-T248, but it's hard to know what's preferable for processing and other things downstream. 
 (survey_name == name for visualization for tracking the scheduler and debugging; 
  observation_reason == goes to the consdb, maybe something for processing;
  target_name == goes to the consdb, maybe something for processing;
  science_program == identifies the JSON BLOCK, goes to nextVisit for `survey`, goes to the consdb;
  scheduler_note == should be going to the consdb, should be returned to the FBS for visit information, don't think either of these are happening yet?)